sotodlib/preprocess/processes.py

```diff
@@ -889,21 +889,28 @@ def save(self, proc_aman, azss_stats):
             proc_aman.wrap(self.calc_cfgs["azss_stats_name"], azss_stats)
 
     def process(self, aman, proc_aman, sim=False):
-        if self.calc_cfgs.get('azss_stats_name') in proc_aman and self.process_cfgs["subtract"]:
-            if sim:
-                tod_ops.azss.get_azss(aman, **self.calc_cfgs)
+        if self.process_cfgs.get("subtract"):
+            if self.calc_cfgs.get('azss_stats_name') in proc_aman:
+                if sim:
+                    tod_ops.azss.get_azss(aman, **self.calc_cfgs)
+                else:
+                    tod_ops.azss.subtract_azss(
+                        aman,
+                        proc_aman.get(self.calc_cfgs.get('azss_stats_name')),
+                        signal=self.calc_cfgs.get('signal', 'signal'),
+                        scan_flags=self.calc_cfgs.get('scan_flags'),
+                        method=self.calc_cfgs.get('method', 'interpolate'),
+                        max_mode=self.calc_cfgs.get('max_mode'),
+                        azrange=self.calc_cfgs.get('azrange'),
+                        in_place=True
+                    )
             else:
-                tod_ops.azss.subtract_azss(
-                    aman,
-                    proc_aman.get(self.calc_cfgs.get('azss_stats_name')),
-                    signal=self.calc_cfgs.get('signal', 'signal'),
-                    scan_flags=self.calc_cfgs.get('scan_flags'),
-                    method=self.calc_cfgs.get('method', 'interpolate'),
-                    max_mode=self.calc_cfgs.get('max_mode'),
-                    azrange=self.calc_cfgs.get('azrange'),
-                    in_place=True
-                )
+                cfgs = dict(copy.deepcopy(self.calc_cfgs))
+                cfgs.pop('subtract_in_place', None)
+                tod_ops.azss.get_azss(aman, subtract_in_place=True, **cfgs)
```

Comment:
Why do we not do `tod_ops.azss.get_azss(aman, **cfgs)` here instead?

Reply:
Because we in this segment of the logic we want to run subtract_in_place=True so if `subtract_in_place` is present in calc_cfgs I'm deleting it and then directly passing subtract_in_place=True to the get_azss fn

Reply:
I'm confused.. We should have subtract_in_place=True is present in cal_clfgs if we want to subtract azss, and passing **self.calc_cfgs will do same thing.

Are you saying that subtract_in_place=False should be overwritten to True if process.subtract True because it's redundunt?

Reply:
Yes, perhaps I should just have the check at the beginning if `process.subtract` and `calc_cfgs['subtract_in_place']` conflict and just raise a ValueError there?

Reply:
Okay, I was not aware of genral rules of proprocess. 
Then I think it's better to change the subtract_in_place part of docstring and change example config to omit subtract_in_place from calc_config and specify subtraction by process config.